### PR TITLE
[zh] Translate Reference/glossary/affinity.md

### LINKED ISSUE #30619: [zh] Translate docs/reference/glossary/affinity.md into Chinese
Hi, I'm interested to translate the "reference/glossary/affinity.md" page.

This is a Feature Request:

translate en/docs/reference/glossary/pod-disruption.md into Chinese.

What would you like to be added:

zh/docs/reference/glossary/affinity.md

/language zh
/kind feature

### PR DESCRIPTION
**Pull request by UOS-liang**
Fixes #30619

### CODE REVIEW

File: content/zh/docs/reference/glossary/affinity.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Affinity
+id: affinity
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
```
Comment (sftim) [inline content/zh/docs/reference/glossary/affinity.md:5]:
```suggestion
full_link: /zh/docs/concepts/scheduling-eviction/assign-pod-node/#%E4%BA%B2%E5%92%8C%E6%80%A7%E4%B8%8E%E5%8F%8D%E4%BA%B2%E5%92%8C%E6%80%A7
```

File: content/zh/docs/reference/glossary/affinity.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+short_description: >
+     调度程序用于确定在何处放置 Pods 的规则
+aka:
```
Comment (sftim) [inline content/zh/docs/reference/glossary/affinity.md:8]: Optionally: add a Chinese equivalent term here (eg: 亲和性).

File: content/zh/docs/reference/glossary/affinity.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+In Kubernetes, _affinity_ is a set of rules that give hints to the scheduler about where to place pods.
+-->
+在 Kubernetes 中，_affinity_ 是一组规则，它向调度程序提示在何处放置 Pods。
+<!--more-->
+<!--
+There are two kinds of affinity:
+-->
+affinity 有两种:
+* [node affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
+* [pod-to-pod affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity)
```
Comment (sftim) [inline content/zh/docs/reference/glossary/affinity.md:36]: I would link to the Chinese page here.

File: content/zh/docs/reference/glossary/affinity.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--
+The rules are defined using the Kubernetes {{< glossary_tooltip term_id="label" text="labels">}},
+and {{< glossary_tooltip term_id="selector" text="selectors">}} specified in {{< glossary_tooltip term_id="pod" text="pods" >}}, 
+and they can be either required or preferred, depending on how strictly you want the scheduler to enforce them.
+-->
+这些规则是使用 Kubernetes {{< glossary_tooltip term_id="label" text="标签（labels）">}} 
+和 {{< glossary_tooltip term_id="pod" text="pods" >}} 中指定的{{< glossary_tooltip term_id="selector" text="调度器（selectors）">}}定义的，
```
Comment (sftim) [inline content/zh/docs/reference/glossary/affinity.md:44]: I'd expect to see
```markdown
这些规则是使用 Kubernetes {{< glossary_tooltip term_id="label" text="label">}} （标签）
和 {{< glossary_tooltip term_id="pod" text="pods" >}}
中指定的{{< glossary_tooltip term_id="selector" text="selector">}} （调度器）定义的，
```
or
```markdown
这些规则是使用 Kubernetes {{< glossary_tooltip term_id="label" text="标签">}} （label）
和 {{< glossary_tooltip term_id="pod" text="pods" >}}
中指定的{{< glossary_tooltip term_id="selector" text="调度器">}} （selector）定义的，
```
but I don't know enough about conventions for localizing to Chinese to be sure that either is right.

**Review by sftim — COMMENTED** [review]
Hi. I don't speak any Chinese but I _have_ reviewed quite a lot of localization PRs. Reviewers for Chinese: do these suggestions make sense?

**Comment by sftim** [comment]
BTW there's a new-style CLA in place but not the old-style one. I'm not sure if we need you to take any extra action @UOS-liang or if we (Kubernetes project) can fix that for you automatically.

**Comment by sftim** [comment]
We'll find out and let you know.

**Review by Arhell — COMMENTED** [review]
Please sign CLA
https://github.com/kubernetes/website/pull/30633#issuecomment-978921221

**Comment by UOS-liang** [comment]
signed

**Comment by RA489** [comment]
> signed

/check-cla

**Comment by tengqm** [comment]
@UOS-liang pls make sure you are commiting the PR using the same email address used for CLA.

**Comment by jimangel** [comment]
Hey there! @UOS-liang, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by UOS-liang** [comment]
close

Resubmit due to CLA error
https://github.com/kubernetes/website/pull/30889

**Pull request closed without merging**